### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/blog/home.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#511] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/blog/home.html

vuepress-theme-hope 允许你启用博客风格的首页。 你需要在相应的页面的 frontmatter 中，设置 home: true 和 layout: BlogHome 来使用博客主页。

https://theme-hope.vuejs.press/zh/guide/blog/home.html

**Comment by WXL570CN** [comment]
我想升级V2.x，但我想要保留V1.x的UI怎么办？

**Reply by WXL570CN** [reply]
个人觉得V2.x的博客风格有点杂乱，不如1.x的简洁

**Reply by Mister-Hope** [reply]
那你就只能停留在 1.x 了，大多数用户都认为 2.x 的 UI 更好。

**Comment by lzc20** [comment]
如何首页不想展示文章如何处理呢

**Reply by Mister-Hope** [reply]
自定义组件 https://theme-hope.vuejs.press/zh/guide/advanced/replace.html

**Comment by DUDU-dudu-DUDU** [comment]
博主大大  如何把博主信息放在左边

**Reply by Mister-Hope** [reply]
我们提供了一个预设，导入即可。https://theme-hope.vuejs.press/zh/guide/customize/presets.html#%E5%8D%9A%E5%AE%A2

**Comment by weiyong996** [comment]
您好，不想显示首页的大图片bgImage，如何设置

**Reply by Mister-Hope** [reply]
- https://theme-hope.vuejs.press/zh/guide/blog/home.html
- https://theme-hope.vuejs.press/zh/config/frontmatter/blog-home.html
- https://theme-hope.vuejs.press/zh/config/frontmatter/project-home.html
- https://theme-hope.vuejs.press/zh/guide/layout/home.html

文档都有

**Comment by GuthubChenJiDian** [comment]
hope帅哥你好，请问一下如何跟你的个人博客网站一样，每次刷新，博客主页的图片都会变化？

**Reply by GuthubChenJiDian** [reply]
我知道了,每次访问这个地址都会返回不同的图片：https://file.mo7.cc/api/public/bz
